docs/getting-started-guides/README.md

```diff
@@ -147,6 +153,7 @@ libvirt              | custom       | Fedora | flannel     | [docs](fedora/flann
 KVM                  | custom       | Fedora | flannel     | [docs](fedora/flannel_multi_node_cluster.md)      |          | Community ([@aveshagarwal](https://github.com/aveshagarwal))
 Mesos/Docker         | custom       | Ubuntu | Docker      | [docs](mesos-docker.md)                           | [✓][4]  | Community ([Kubernetes-Mesos Authors](https://github.com/mesosphere/kubernetes-mesos/blob/master/AUTHORS.md))
 Mesos/GCE            |              |        |             | [docs](mesos.md)                                  |          | Community ([Kubernetes-Mesos Authors](https://github.com/mesosphere/kubernetes-mesos/blob/master/AUTHORS.md))
```

Comment:
The mesos guide doesn't come with automated cluster deployment scripts. It describes a manual process. So there's no config management, operating system, or networking specified. They could all be N/A, but we left them blank instead.